controllers/v1/admin.js

```
@@ -123,4 +123,51 @@ module.exports = class Admin {
+						message: CONSTANTS.apiResponses.ADMIN_TOKEN_MISSING_MESSAGE,
+					}
+				}
+				return resolve(deletedEntity)
+			} catch (error) {
+				return reject({
+					status: error.status || HTTP_STATUS_CODE.internal_server_error.status,
+					message: error.message || HTTP_STATUS_CODE.internal_server_error.message,
+					errorObject: error,
+				})
+			}
+		})
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)